ports/nrf/mpconfigport.h

```diff
@@ -38,6 +38,11 @@
 #define MICROPY_PY_UBINASCII                     (1)
 #define MICROPY_PY_UJSON                         (1)
 
+#ifndef BOARD_HAS_32KHZ_XTAL
+// Assume crystal is present, which is the most common case.
+#define BOARD_HAS_32KHZ_XTAL (0)
```

Comment:
the comment seem to contradict the value

Reply:
RIghto :face_with_head_bandage: